ci/scripts/bazel-tests.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e -x
+
+# This doesn't install dependencies, but should otherwise behave the
+# same as what CI would do on a pull request
+DEFAULT="master"
+
+# use tests from a condonedlist, if a target is moved or disabled, it should
+# also be moved in the following list in order for CI to pass.
```

Comment:
I'd rather have the set of disqualified targets encoded into the test rules themselves rather than having to maintain a list like this.

We could, for example, apply a `tags = ["broken"]` to tests that are broken _and_ that we temporarily accept as broken.  In that case, the CI test execution command can be something like:
```
bazel test //... --test_tag_filters=-broken
```

And some auditing script can do something like the following to enumerate known broken tests:
```
bazel query 'attr("tags", "broken", tests(//...))'
```

The current solution is ok for now, but we need to get CI to the point where we're just unconditionally executing all tests and/or reasonable subsets of tests based on queries rather than hand-maintained lists.

Reply:
I need to query what's intentionally broken at PR head and save it when I rerun at master, but if we exclude it from running at master than we wont see it pass there.